doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -54,6 +57,180 @@ Additionally, maintaining ABI compatibility allows the nRF54H20 SoC binary compo
+* System Controller Firmware (SCFW) releases are now included in the Secure Domain Firmware (SDFW).
+  See :ref:`scfw_5_0_0` for details.
+
+Updated
+-------
+
+* The CPUCONF service request definition.
+* The CPU and WAIT parameters are now both packed into the first 4-byte value, and the message data is sent inline in the request.
+* Support for initializing a subset of global domain peripherals by configuring ``UICR.PERIPHCONF``. This enables the initial configuration of the CTRLSEL GPIO pin, global IRQ mapping, IPC mapping, global PPIBs, and more.
+* ``UICR.PERIPHCONF`` reintroduces functionality that was previously available with specialized ``UICR.*`` registers, but with a lower-level interface that is more powerful, flexible, and future proof.
+
+.. _scfw_5_0_0:
+
+System Controller Firmware (SCFW) v5.0.0
+----------------------------------------
+
+Added
+~~~~~
+
+* SWEXT service.
+
+Updated
+~~~~~~~
+
+* Reduced MRAM auto power down timeout (helps with lower power usage).
+* GRCCONF module code optimization.
+* IPC communication code optimization (Zephyr IPC service used directly without additional queue).
+* Fixed higher power consumption when clock switcher changed to LFRC.
+
+Secure Domain Firmware (SDFW) v21.0.1
+=====================================
+
+Added
+-----
+
+* NCSDK-33583: Boot report is now written to radio core.
+
+Updated
+-------
+
+* Enable link-time optimization.
+* NCSDK-32173: Disable CRACEN microcode loading.
+
+Fixed
+-----
+
+* NCSDK-33306: The application core is now started in halted mode when IRONside detects errors in the UICR or BICR. This ensures that it is possible to recover from these errors by writing proper values with a debugger.
+
+Secure Domain Firmware (SDFW) v21.0.0
+=====================================
+
+Added
+-----
+
+* NCSDK-32168: Added support for the IRONside SE update service.
+* NCSDK-32444: Added experimental support for a new UICR format. At this stage, the functionality is mainly for internal testing and development, and user tools for interacting with UICR will be added at a later stage.
+* NCSDK-32393: Added boot report support.
+* NCSDK-32925: Added CPUCONF service for booting the radio core. Only ``hello world`` is currently supported.
+* NCSDK-32441: Added IRONside calls, the successor to SSF.
+
+Updated
+-------
+
+* NCSDK-32912: The limited PSA service is now implemented as an IRONside call. This replaces the temporary IPC mechanism from the last release.
+
+Fixed
+-----
+
+* NCSDK-33265: Fixed an issue that made it so the ``CTRLAP.BOOTSTATUS`` firmware sequence number was always set to zero.
+
+Secure Domain Firmware (SDFW) v20.1.0
+=====================================
+
+Added
+-----
+
+* NCSDK-32163: Added experimental support for limited PSA service. This is built on top of a temporary IPC mechanism which will soon be replaced. The top-level interface will remain the same.
```

Comment:
By "PSA service", I guess you mean PSA Crypto API? Or maybe, more specifically, Oberon PSA Crypto? Please update. "PSA service" is generic and incorrect.